src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,10 @@
 public class Main {
 
     public static void main(String[] args) {
+        Person person = new Person("Jan", "Kowalski", 25, "1234567890");
+        System.out.println(person);
+        person.setAge(-10);
+        System.out.println(person);
```

Comment:
To powinno być w bloku try-catch. Powinnaś złapać potencjalne wyjątki IncorrectAgeException lub NameUndefinedException i w bloku catch możesz wtedy wyświetlić czytelny komunikat.